activities_sticky_queues/tasks.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import random
+from dataclasses import dataclass
+from datetime import timedelta
+from hashlib import md5
+from pathlib import Path
+from typing import Any, Callable, Coroutine, List
+
+from temporalio import activity, workflow
+
+TIME_DELAY = 3.0
+LOCAL_PATH = Path(__file__).parent / "demo_fs"
+
+
+@dataclass
+class DownloadObj:
+    url: str
+    path: str
+
+
+async def _get_available_task_queue(available_queues: List[str]) -> str:
+    """Randomly assign the job to a queue"""
+    return random.choice(available_queues)
+
+
+@activity.defn
+async def download_file_to_worker_filesystem(details: DownloadObj):
```

Comment:
```suggestion
async def download_file_to_worker_filesystem(details: DownloadObj) -> None:
```

Would be considered a bit pedantic for most Python functions, but explicit return type is a good practice on activities for caller and return value serialization reasons.